File: SDKs/googleapis.com/area120tables/v1alpha1/python/src/sdk/models/shared/listrowsresponse.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class ListRowsResponse:
    r"""ListRowsResponse
    Response message for TablesService.ListRows.
    """
    
    next_page_token: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('nextPageToken') }})
    rows: Optional[List[Row]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('rows') }})
    

File: SDKs/googleapis.com/area120tables/v1alpha1/typescript/src/sdk/models/shared/listrowsresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Row } from "./row";



// ListRowsResponse
/** 
 * Response message for TablesService.ListRows.
**/
export class ListRowsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=nextPageToken" })
  nextPageToken?: string;

  @SpeakeasyMetadata({ data: "json, name=rows", elemType: Row })
  rows?: Row[];
}
